scripts/check_delta.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2020 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+Get different tests delta between two xml files,
+for example, two xml files of same board with different commits,
+or two xml files of same commit on different boards
+"""
+
+import argparse
+import os
+import sys
+import xml.etree.ElementTree as ET
+
+CWD = os.getcwd()
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+            description = "find different tests delta")
+    parser.add_argument("-x", "--xml-file", default = None,
+            action = "append", help = "xml file path. This option "
+            "may be used multiple times.")
```

Comment:
"This option can be used two times"
Seems that we can compare only 2 files, yes?

Reply:
fixed.